app/src/main/java/com/fstyle/androidtrainning/screen/main/playingfragments/lyrics/LyricsFragment.java

```diff
@@ -1,23 +1,52 @@
 package com.fstyle.androidtrainning.screen.main.playingfragments.lyrics;
 
+import android.content.BroadcastReceiver;
+import android.content.Context;
+import android.content.Intent;
+import android.content.IntentFilter;
+import android.os.Build;
```

Comment:
ERROR: Unused import: android.os.Build. Use Ctrl+Shift+O (Eclipse) or Ctrl+Alt+O (Intellij) to sort imports